web-registry/src/types/project/project.ts

```diff
@@ -0,0 +1,51 @@
+export type VcsProjectMetadata = {
```

Comment:
you already have this built so not sure this would be helpful for you, but for my story I ran the JSON structures through a basic type generator and used them. Just makes it so you don't have to modify as much (moving forward, not suggesting you change this). see: https://github.com/regen-network/regen-web/pull/866/files#diff-557a5b59cd7201c768fefa749405991354b999f4b85d50d79e9e00f1efda2043

Reply:
Looks good. In this case, I have to temporarily deviate from the standard, so this is a helper for now. I will leave a TODO to update once the standard is finished.